onnxscript/function_libs/torch_aten/tensor_typing.py

```diff
@@ -39,6 +43,9 @@
     INT32,
     INT64,
     UINT8,
+    UINT16,
```

Comment:
Any reason why we add these unsigned types?

Reply:
Following the spec: https://github.com/onnx/onnx/blob/main/docs/Changelog.md#ConcatFromSequence-11

Reply:
We didn't add these types and had above comments based on [this](https://pytorch.org/docs/stable/tensors.html). Probably this is not necessary, but anyway , it doesn't hurt.

Reply:
I see. I will revert it back